scripts/build.sh

```diff
@@ -1,12 +1,58 @@
 #!/bin/bash
+# Single source of truth for building Firecloud-Ui.
+# @ Jackie Roberti and Isaac Zarsky
+#
+# Provide command line options to do one or several things:
+#   compile: compile clojure proj into /target
+#   -d | --docker : provide arg either "build" or "push", to build and push docker image
+# Jenkins build job should run with all options, for example,
+#   ./docker/build.sh compile -d push
```

Comment:
1. Will we ever _not_ want to compile? Why?
2. Why do we need the `-d` flag before `push` or `build`?
3. Should we set some help text to echo out if there are no arguments?